noir/noir-repo/test_programs/execution_success/verify_honk_proof/src/main.nr

```diff
@@ -1,11 +1,12 @@
 
 // This circuit aggregates a single Honk proof from `assert_statement_recursive`.
+global SIZE_OF_PROOF_IF_LOGN_IS_28 : u32 = 409;
```

Comment:
super nit: find a way to use RECURSIVE_PROOF_LENGTH from constants.nr.

Reply:
The constant in constants.nr must be for Plonk? Cant use this there. I'm not sure how I could connect this SIZE_OF_PROOF_IF_LOGN_IS_28 between rust and cpp, thats beyond me. I'll add a comment to the bberg constant though

Reply:
It looks like the constant there is for plonk..